test/Passes/tpp-identity.mlir

```diff
@@ -0,0 +1,63 @@
+// RUN:  tpp-opt %s  --default-tpp-passes | tpp-run -e entry  -entry-point-result=void | FileCheck %s
+
+func.func @pack1(%in: tensor<4x4xf32>, %out: tensor<2x2x2x2xf32>) ->  tensor<2x2x2x2xf32> {
+  %1 = tensor.pack %in inner_dims_pos = [0, 1] inner_tiles = [2,2] into %out : tensor<4x4xf32> -> tensor<2x2x2x2xf32>
+  return %1 : tensor<2x2x2x2xf32>
+}
+
+func.func @pack2(%0: tensor<1x2x2x4xf32>, %1:  tensor<1x2x2x2x2xf32>)-> tensor<1x2x2x2x2xf32>{
+ %2 = tensor.pack %0  outer_dims_perm = [0, 3, 1, 2] inner_dims_pos = [3] inner_tiles = [2] into %1 : tensor<1x2x2x4xf32> -> tensor<1x2x2x2x2xf32>
+  return %2: tensor<1x2x2x2x2xf32>
+}
+
+func.func @pack3(%in: tensor<8x2x2x2xf32>, %out: tensor<2x2x1x4x2x2xf32>)-> tensor<2x2x1x4x2x2xf32>{
+  %2 = tensor.pack %in outer_dims_perm = [3, 2, 1, 0] inner_dims_pos=[1, 0] inner_tiles = [2, 2] into %out:   tensor<8x2x2x2xf32>->tensor<2x2x1x4x2x2xf32>
+  return %2: tensor<2x2x1x4x2x2xf32>
+}
+
+
+func.func @entry(){
+  %0 = arith.constant dense<[[0.0, 1.0, 2.0, 3.0],
+                               [4.0, 5.0, 6.0, 7.0],
+                               [8.0, 9.0, 10.0, 11.0],
+                               [12.0, 13.0, 14.0, 15.0]]> : tensor<4x4xf32>
+  %1 = tensor.empty():tensor<2x2x2x2xf32>
+  %c0 = arith.constant 0 : index
+  %2 = call @pack1(%0,%1) : (tensor<4x4xf32>, tensor<2x2x2x2xf32>) -> tensor<2x2x2x2xf32>
+  %d1 = arith.constant -1.0 : f32
+  %v0 = vector.transfer_read %1[%c0, %c0, %c0, %c0], %d1 : tensor<2x2x2x2xf32>, vector<2x2x2x2xf32>
+  vector.print %v0 : vector<2x2x2x2xf32> 
+  // CHECK: ( ( ( ( 0, 1 ), ( 4, 5 ) ), ( ( 2, 3 ), ( 6, 7 ) ) ), ( ( ( 8, 9 ), ( 12, 13 ) ), ( ( 10, 11 ), ( 14, 15 ) ) ) ) 
+  
+  %3 = arith.constant dense <[[[[0.0, 1.0, 2.0, 3.0],  [4.0, 5.0, 6.0, 7.0]],
+                                [[8.0, 9.0, 10.0, 11.0],[12.0, 13.0, 14.0, 15.0]]]]>: tensor<1x2x2x4xf32>
+  %4 = tensor.empty():tensor<1x2x2x2x2xf32>
+  %5 = call @pack2(%3, %4):(tensor<1x2x2x4xf32>,  tensor<1x2x2x2x2xf32>)->(tensor<1x2x2x2x2xf32>)
```

Comment:
It is a bit confusing to have to look up to the `pack` function to see what pack is being done, here. I think it's easier to just have the pack operation here and not a function call.